### PR TITLE
Update Polymesh Genesis Hash

### PR DESCRIPTION
**Pull request by adamdossa**
Updates to our latest mainnet hash - needed for h/w wallet integration etc.

### CODE REVIEW

**Comment by adamdossa** [comment]
@jacogr please could I request a review - hopefully a simple change.

**Review by jacogr — COMMENTED** [review]
Dangit... my comment was marked pending, for days now :(

File: packages/networks/src/defaults.ts [hunk 1]
```diff
@@ -58,7 +58,7 @@ export const knownGenesis: KnownGenesis = {
     '0x91b171bb158e2d3848fa23a9f1c25182fb8e20313b2c1eb49219da7a70ce90c3'
   ],
   polymesh: [
-    '0x9deeb940c92ae02111c3bd5baca89970384f4c9849f02a1b2e53e66414d30f9f'
+    '0x6fbd74e5e1d0a61d52ccfe9d4adaed16dd3a7caa37c6bc4d0c2fa12e8b2f4063'
```
[the remark this answers is not in the document]
Comment (adamdossa) [reply packages/networks/src/defaults.ts:61]: Hey @jacogr - in our case this isn't needed. The previous hash was for a short-lived (~3 months) incentivised test network which has now been closed. The new hash is for our mainnet which is now live!

**Review by jacogr — APPROVED** [review]
Once again, sorry for the delay, my comments were stuck in pending (problem between keyboard and chair on this side) and didn't want to merge before.

**Pull request merged**